public/app/diarization-timeline.js

```
@@ -0,0 +1,457 @@
+    ctx.fillText(_fmtTime(t), x + 2, HEADER_HEIGHT - 2);
+  }
+}
+
+// ─────────────────────────────────────────────
+// Interaction
+// ─────────────────────────────────────────────
+
+function _bindCanvasClick() {
+  Timeline.canvas.addEventListener('click', (e) => {
+    const rect = Timeline.canvas.getBoundingClientRect();
+    const x    = e.clientX - rect.left;
```

ESLint

[error] 'x' is assigned a value but never used.

(no-unused-vars)